scripts/events.py

```diff
@@ -551,7 +551,7 @@ def handle_lost_cats_return(self):
         lost_cat = None
         for cat in Cat.outside_cats.values():
             if cat.outside and cat.status not in [
-                    'kittypet', 'loner', 'rogue', 'former clancat'
+                    'kittypet', 'loner', 'rogue', 'former Clancat'
```

Comment:
Is 'former Clancat' a valid status?
I thought 'exiled' is used for ex-Clancats.